PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Views/PlayerViewController.swift

```diff
@@ -9,23 +9,55 @@ import UIKit
 import AVFoundation
 import MediaPlayer
 
-class PlayerViewController: OOOoyalaPlayerViewController {
+//now inited from I.B.
+class PlayerViewController: UIViewController {
   
+  // MARK: - Private properties
   // properties for the video
   public var option: PlayerSelectionOption!
   
   // properties required for a Fairplay asset
   private var apiKey: String?
   private var apiSecret: String?
+  private var ooyalaPlayerVC: OOOoyalaPlayerViewController!
```

Comment:
Does it really need to be an implicit unwrapped optional?

Reply:
Description of initialization method says that result is not-optional, that's why I used non-optional declaration. 
_Declaration: init!(player: OOOoyalaPlayer!)_

Should I avoid this even when init is not-optional?

Reply:
Changed to optional